clang/include/clang/Driver/Options.td

```diff
@@ -9469,10 +9469,10 @@ def fdx_rootsignature_version :
   Group<dxc_Group>,
   Visibility<[ClangOption, CC1Option]>,
   HelpText<"Root Signature Version">,
-  Values<"rootsig_1_0,rootsig_1_1">,
+  Values<"rootsig_1_0,rootsig_1_1,rootsig_1_2">,
   NormalizedValuesScope<"llvm::dxbc::RootSignatureVersion">,
-  NormalizedValues<["V1_0", "V1_1"]>,
-  MarshallingInfoEnum<LangOpts<"HLSLRootSigVer">, "V1_1">;
+  NormalizedValues<["V1_0", "V1_1", "V1_2"]>,
+  MarshallingInfoEnum<LangOpts<"HLSLRootSigVer">, "V1_2">;
```

Comment:
I think we may want to keep the default to v1.1 to match what dxc emits. We should discuss this.